OTPKit/Features/OriginDestination/Sheets/OriginDestinationSheetView.swift

```diff
@@ -11,12 +11,10 @@ public struct OriginDestinationSheetView: View {
     @Environment(\.dismiss) var dismiss
     @EnvironmentObject var sheetEnvironment: OriginDestinationSheetEnvironment
 
-    @StateObject private var locationService = LocationService()
+    @ObservedObject private var locationManagerService = LocationManagerService.shared
```

Comment:
I put the LocationManagerService as singleton and make it as ObservedObject? What do you think of this approach? @aaronbrethorst